docs/integrations/saas-cloud/asana.md

```diff
@@ -0,0 +1,94 @@
+
+Key features of the Asana App include:
+- **Pre-built dashboard**. You may view the most important metrics connected to Asana data quickly using this app's pre-built dashboard. These dashboards offer a high-level overview of user updates, data exports, role changes, and logins.
+- **Customizable searches**. With the help of this app, you may build custom searches to examine particular Asana data elements, allowing you to go deeper into the data and find insights that are pertinent to your company's needs.
+- **Real-time monitoring**. By providing real-time monitoring of Asana data, this tool enables you to keep track of the most recent developments and trends in your team's work.
+- **Integration with other sources**. This app can be used in conjunction with other Sumo Logic Apps, which enables you to gain a more holistic view of the data.
+
+## Log Types
+
+This App uses Asana Source to collect [Audit Logs](https://developers.asana.com/reference/audit-log-api) from Asana.
+
+### Sample log message
+
+```json title="Audit Log"
+{
+ "gid": "12345",
+ "actor": {
+   "actor_type": "user",
+   "email": "gregsanchez@example.com",
+   "gid": "1111",
+   "name": "Greg Sanchez"
+ },
+ "context": {
+   "api_authentication_method": "cookie",
+   "client_ip_address": "1.1.1.1",
+   "context_type": "web",
+   "oauth_app_name": "string",
+   "user_agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.103 Safari/537.36"
+ },
+ "created_at": "2021-01-01T00:00:00.000Z",
+ "details": {},
+ "event_category": "deletion",
+ "event_type": "task_deleted",
+ "resource": {
+   "email": "string",
+   "gid": "1111",
+   "name": "Example Task",
+   "resource_subtype": "milestone",
+   "resource_type": "task"
+ }
+}
+```
+
+### Sample Query
+
+```sql title="Events by App Authorization"
+_sourceCategory="asana-app"
+| json "gid","event_type","resource.name","resource.email","resource.resource_type","event_category", "created_at", "actor.name", "actor.email","context.client_ip_address" as gid, event_type, resource_name, resource_email, resource_type, event_category, created_at, actor_name, actor_email, ip nodrop
+| where event_type in ("user_app_authorized","user_app_revoked")
+| count_distinct(gid) as frequency by event_type
+| sort by frequency
+```
+
+## Set up collection
+
+To set up [Cloud to Cloud Integration Asana Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/asana-source/) for the Asana App, follow the instructions provided. These instructions will guide you through the process of creating a source using the Asana Source category, which you will need to use when installing the app. By following these steps, you can ensure that your Asana App is properly integrated and configured to collect and analyze your Asana data.
+
+## Installing the Asana App
```

Comment:
```suggestion
## Installing the Asana app
```

Reply:
Please maintain consistency throughout the doc, "app" should be written in lowercase. Please make the changes in other instances as well.

Reply:
We should not capitalize unless it appears in the UI.